src/firefly/js/voAnalyzer/VoDataLinkServDef.js

```diff
@@ -97,7 +97,7 @@ function getSDDescription(table, ID) {
     const descriptionCol = getColumnIdx(table, 'description');
     if (descriptionCol === -1 || serviceDefCol === -1) return;
     if (table.totalRows > 50) return;
-    const sdRow = table.tableData.data.find((dAry) => dAry[serviceDefCol] === ID);
+    const sdRow = table?.tableData?.data?.find((dAry) => dAry[serviceDefCol] === ID);
```

Comment:
addd this check @robyww because doing a column filter search on the data products table was crashing the app if the result returned nothing